lte/gateway/c/oai/tasks/mme_app/mme_app_state_manager.cpp

```diff
@@ -136,6 +136,9 @@ void MmeNasStateManager::mme_nas_state_init_local_state() {
         mme_statistic_timer_);
     state_cache_p->statistic_timer_id = 0;
   }
+  if (is_initialized) {
+    mme_app_recover_timers_for_all_ues();
```

Comment:
Is this code ever called, as is_initialized is set to false when mme_nas_state_init_local_state is called

Reply:
Yes Shruti, we don't really need this statement. I shall remove and check-in